games-engines/instead/instead-3.3.3.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="cs de en es fr it nl pt ru uk"
+PLOCALE_BACKUP="en"
+LUA_COMPAT=( lua5-1 luajit )
+
+inherit cmake lua-single l10n
+
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/instead-hub/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/instead-hub/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="INSTEAD text-based quest game engine"
+HOMEPAGE="http://instead3.syscall.ru/"
```

Comment:
Please try https for HOMEPAGE

Reply:
fixed for both `3.3.3`&`9999`, amended and force-pushed